### DISCUSSION TITLE
Shared base with password [May 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2063] [Discord Threads]
> 🙋‍♂️ **solardesalination**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### we do not have it at the moment but you can raise a feature request with details please  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/970636257669550110)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
